src/main/java/org/entando/plugins/pda/controller/task/TaskCommentController.java

```diff
@@ -0,0 +1,90 @@
+package org.entando.plugins.pda.controller.task;
+
+import static org.entando.plugins.pda.controller.AuthPermissions.TASK_COMMENTS_CREATE;
+import static org.entando.plugins.pda.controller.AuthPermissions.TASK_COMMENTS_DELETE;
+import static org.entando.plugins.pda.controller.AuthPermissions.TASK_COMMENTS_GET;
+import static org.entando.plugins.pda.controller.AuthPermissions.TASK_COMMENTS_LIST;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.entando.keycloak.security.AuthenticatedUser;
+import org.entando.plugins.pda.core.engine.Connection;
+import org.entando.plugins.pda.core.engine.Engine;
+import org.entando.plugins.pda.core.model.Comment;
+import org.entando.plugins.pda.core.request.CreateCommentRequest;
+import org.entando.plugins.pda.engine.EngineFactory;
+import org.entando.plugins.pda.service.ConnectionService;
+import org.entando.web.response.SimpleRestResponse;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@Api(tags = "TaskComment")
+@RequestMapping(path = "/connections/{connId}/tasks")
```

Comment:
This can be `/connections/{connId}/tasks/{taskId}/comments` and remove `/{taskId}/comments` from the methods.

Reply:
Yeah, totally makes sense.